pages/linux/hwinfo.md

```diff
@@ -3,17 +3,17 @@
 > Probe for the hardware present in the system.
 > More information: <https://manpages.opensuse.org/hwinfo/hwinfo.8.en.html>.
```

Comment:
seems to be a bit of history behind this link (check commit history :eyes:)

Reply:
I do remember that PR. Lots of drama.